src/components/DevNavMenu.tsx

```diff
@@ -712,7 +712,7 @@ DevNavMenuTuts.set('cumulusTutorial', [
         link: '/tutorials/v3/cumulus/start-relay',
       },
       {
-        title: `Connect to a Parachain`,
+        title: `Connect a Parachain`,
```

Comment:
`to` is intentionally removed from the title. Because the full sentence is `Connect a Parachain to a relay chain`, not connecting to a parachain.